### PR TITLE
create the community page

### PR DESCRIPTION
**Pull request by tindt2211**

### CODE REVIEW

File: src/css/community/page.module.css [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+  border-top: 1px solid #FFFFFF;
+  opacity: 0.1;
+}
+
+.padding_60 {
+  padding: 60px;
+}
+
+.defalut_text {
+  font-family: 'Inter';
```
Comment (bsmth) [inline src/css/community/page.module.css:51]: Overriding the font family here with `Inter` overrides the global font used across other pages and this page. There's a mix of sans serif and serif, looks like we can remove the `font-family` properties from this module

File: src/css/community/page.module.css [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+  margin: 30px 0;
+  border-top: 1px solid #FFFFFF;
+  opacity: 0.1;
+}
+
+.padding_60 {
+  padding: 60px;
+}
+
+.defalut_text {
```
Comment (bsmth) [inline src/css/community/page.module.css:50]: typo default

File: src/css/community/page.module.css (hunk 1 above)
Comment (tindt2211) [reply src/css/community/page.module.css:51]: @bsmth 
Should I remove font-family from this module?

File: src/pages/community.tsx [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+
+          <p
+            className={clsx(
+              seCss.section__subtitle,
+              seCss["section__subtitle--jumbotron"],
+              seCss["section__subtitle--accent"],
+              paCss.section__subtitle,
+            )}
+          >
+            Maecenas sed diam eget risus varius blandit sit amet non magna.
```
Comment (bsmth) [inline src/pages/community.tsx:63]: lorem ipsum leftover

File: src/css/community/page.module.css (hunk 2 above)
Comment (tindt2211) [reply src/css/community/page.module.css:50]: I will update it.

File: src/pages/community.tsx [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+            <p
+              className={clsx(seCss.section__subtitle, paCss.section__subtitle)}
+            >
+              As a way of saying &quot;thank you&quot; for being part of the
+              journey, we want to offer contributors some of our coveted
+              stickers, pins, t-shirts, and awesome virtual swag.
+            </p>
+            <div className={paCss.border} />
+            <div>
+              <p className={paCss.defalut_text}>
```
Comment (bsmth) [inline src/pages/community.tsx:93]: I might be wrong, but I don't think we need a class on these `<p>` elements, I think we have a style defined for this in the section CSS module

File: src/pages/community.tsx (hunk 3 above)
Comment (tindt2211) [reply src/pages/community.tsx:63]: I don't understand.
Could you comment more clearly?

File: src/pages/community.tsx [hunk 5]
```diff
@@ -0,0 +1,296 @@
+import React from "react"
+import clsx from "clsx"
+import PageLayout from "@theme/PageLayout"
+import Button from "@theme/Button"
+
+import seCss from "../css//section.module.css"
```
Comment (bsmth) [inline src/pages/community.tsx:6]: minor typo `import seCss from "../css/section.module.css"`

Comment (tindt2211) [reply src/pages/community.tsx:6]: ok, I got it.

File: src/pages/community.tsx [hunk 6]
```diff
@@ -0,0 +1,296 @@
...
+            </div>
+            <div className={paCss.half__section}>
+              <p className={paCss.level__title}>Level 1</p>
+              <h2 className={paCss.section__title}>Show the love</h2>
+              <p className={paCss.defalut_text}>
+                To claim your swag for the this level:
+              </p>
+              <p className={paCss.property}>
+                You have joined our{" "}
+                <a className={paCss.link_item} href="https://slack.questdb.io/">
```
Comment (bsmth) [inline src/pages/community.tsx:136]: the URL can be replaced with `{@slackUrl@}` defined in https://github.com/questdb/questdb.io/blob/master/docusaurus.config.js#L21, see https://github.com/questdb/questdb.io/blame/master/blog/2021-04-20-questdb-release-6-0-alpha.md#L119-L121

File: src/pages/community.tsx (hunk 4 above)
Comment (tindt2211) [reply src/pages/community.tsx:93]: I use several p elements for this page.
p elements have different styles, so I use the different classname.

File: src/pages/community.tsx [hunk 7]
```diff
@@ -0,0 +1,296 @@
...
+                  Community Slack
+                </a>
+              </p>
+              <p className={paCss.property}>
+                You have{" "}
+                <a
+                  className={paCss.link_item}
+                  href="https://github.com/questdb/questdb"
```
Comment (bsmth) [inline src/pages/community.tsx:144]: same as the Slack URL, I think we can use a variable for this `githubUrl`

File: src/pages/community.tsx [hunk 8]
```diff
@@ -0,0 +1,296 @@
...
+            <p className={paCss.level__title}>Level 2</p>
+            <h2 className={paCss.section__title}>Contributor</h2>
+            <p className={paCss.defalut_text}>
+              To claim your swag for the this level:
+            </p>
+            <p className={paCss.property}>
+              You have asked or answered a question on Stack Overflow{" "}
+              <a
+                className={paCss.link_item}
+                href="https://stackoverflow.com/questions/tagged/questdb"
```
Comment (bsmth) [inline src/pages/community.tsx:180]: same here, see `stackoverflowUrl`

File: src/pages/community.tsx (hunk 6 above)
Comment (tindt2211) [reply src/pages/community.tsx:136]: ok, I got it

File: src/pages/community.tsx (hunk 7 above)
Comment (tindt2211) [reply src/pages/community.tsx:144]: ok, I got it

File: src/pages/community.tsx (hunk 8 above)
Comment (tindt2211) [reply src/pages/community.tsx:180]: ok, I got it

File: src/css/community/page.module.css (hunk 1 above)
Comment (bsmth) [reply src/css/community/page.module.css:51]: Yeah, I think that should fix it. I have not built this locally, cannot confirm, please check if this makes the font look uniform after you remove it 👍🏻

File: src/pages/community.tsx (hunk 3 above)
Comment (bsmth) [reply src/pages/community.tsx:63]: This is placeholder text from an older design version, please see the discussion with Nic for the most recent one

File: src/pages/community.tsx (hunk 4 above)
Comment (bsmth) [reply src/pages/community.tsx:93]: OK

File: src/pages/community.tsx (hunk 3 above)
Comment (tindt2211) [reply src/pages/community.tsx:63]: ok

File: src/pages/community.tsx [hunk 9]
```diff
@@ -0,0 +1,315 @@
...
+              <a
+                className={paCss.link_item}
+                href={siteConfig.customFields.stackoverflowUrl}
+              >
+                with the QuestDB tag
+              </a>
+            </p>
+            <p className={paCss.property}>
+              You have voted, commented on, or opened a{" "}
+              <a className={paCss.link_item} href="#">
```
Comment (bsmth) [inline src/pages/community.tsx:205]: Should point to https://github.com/questdb/questdb, feel free to reuse the variable

File: src/pages/community.tsx [hunk 10]
```diff
@@ -0,0 +1,315 @@
...
+    <div>
+      <input
+        className={paCss.custom_input}
+        ref={(node) => (email = node)}
+        type="email"
+        placeholder="Email address"
+      />
+      <Button onClick={submit} className={paCss.signup_button}>
+        Sign Up
+      </Button>
```
Comment (bsmth) [inline src/pages/community.tsx:42]: There's a minor layout issue here where the button wraps under the input element when resizing the page:

**1085px width:**
image

**< 1085px** page width is above the mobile breakpoint:

image

**Review by bsmth — APPROVED** [review]
Thanks @tindt2211 👍🏻

**Pull request merged**